core/backend/src/test/standalone/IModelHost.test.ts

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { assert, expect } from "chai";
+import * as path from "path";
+import { IModelHost, IModelHostConfiguration, KnownLocations } from "../../IModelHost";
+import { BriefcaseManager } from "../../BriefcaseManager";
+import { RpcRegistry } from "@bentley/imodeljs-common";
+import { IModelTestUtils } from "../IModelTestUtils";
+import { Schemas } from "../../Schema";
+
+describe("IModelHost", () => {
+
+  it("valid default configuration", async () => {
+    await IModelTestUtils.shutdownBackend();
+    await IModelHost.startup();
+
+    // Valid registered implemented RPCs
+    expect(RpcRegistry.instance.implementationClasses.size).to.equal(7);
+    expect(RpcRegistry.instance.implementationClasses.get("IModelReadRpcInterface")).to.exist;
+    expect(RpcRegistry.instance.implementationClasses.get("IModelTileRpcInterface")).to.exist;
+    expect(RpcRegistry.instance.implementationClasses.get("IModelWriteRpcInterface")).to.exist;
+    expect(RpcRegistry.instance.implementationClasses.get("SnapshotIModelRpcInterface")).to.exist;
+    expect(RpcRegistry.instance.implementationClasses.get("WipRpcInterface")).to.exist;
+    expect(RpcRegistry.instance.implementationClasses.get("DevToolsRpcInterface")).to.exist;
+    expect(RpcRegistry.instance.implementationClasses.get("Editor3dRpcInterface")).to.exist;
+
+    expect(Schemas.getRegisteredSchema("BisCore")).to.exist;
+    expect(Schemas.getRegisteredSchema("Generic")).to.exist;
+    expect(Schemas.getRegisteredSchema("Functional")).to.exist;
+
+    // Restore the backend to the initial state.
+    await IModelTestUtils.shutdownBackend();
+    await IModelTestUtils.startBackend();
```

Comment:
Shouldn't this cleanup be done in an `after` to avoid cascading failures?

Reply:
I was debating the best way to handle it as each test in this describe would probably wan their own setup.  What do you think about moving the restart to be afterEach, that way each one is back to the original state?